react-with-graphql-tutorial.md

```diff
@@ -178,17 +179,19 @@ query ($organization: String!) {
 }
 {{< /highlight >}}
 
-It defines the `organization` argument as a variable by using the `$` sign. In addition, the type of the argument is provided by defining it as `String` type. Since the argument is required to fulfil the query, the `String` type has an exclamation point. In the next step, in your "Query Variables" panel in GraphiQL, the variables would have the following content for providing the `organization` variable as argument for the query:
+It defines the `organization` argument as a variable using the `$` sign. Also, the argument's type is definined as a `String`. Since the argument is required to fulfil the query, the `String` type has an exclamation point. 
+
+In the "Query Variables" panel, the variables would have the following content for providing the `organization` variable as argument for the query:
 
 {{< highlight javascript >}}
 {
   "organization": "the-road-to-learn-react"
 }
 {{< /highlight >}}
 
-Variable(s) can be used to create dynamic queries when it comes to the implementation in your code. Following this best practice in GraphQL, you don't need to do any manual string interpolation yourself to structure a dynamic query later on. You would provide the query which uses variables as arguments instead and provide the variables when you actually perform the query as request to your GraphQL API. You will see both implementations later in your React application.
+Essentially, ariables can be used to create dynamic queries. Following the best practices in GraphQL, we don't need manual string interpolation to structure a dynamic query later on. Instead, we provide a query that uses variables as arguments, which are available when the query is sent as a request to the GraphQL API. You will see both implementations later in your React application. 
```

Comment:
"Essentially, ariables can be used to ..." to "Essentially, variables can be used to ..."